lib/hound/helpers/log.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Hound.Helpers.Log do
+  @moduledoc "Functions to work with the log"
+
+  import Hound.RequestUtils
+
+  @doc """
+  Fetches console.log() from the browser as a single string of log-lines.
+  """
+  def fetch_log() do
+    fail_if_webdriver_selenium("fetch_log()")
+    
+    get_browser_log()
+    |> Enum.map(&(Map.get(&1, "message")))
+    |> Enum.join("\n")
+  end
+
+  @doc """
+  Fetches all console.log() lines of type 'WARINING' and CRITICAL, from the browser as a single string of log-lines.
+  """
+  def fetch_errors() do
+    fail_if_webdriver_selenium("fetch_errors()")
+    
+    get_browser_log()
+    |> Enum.filter(&(is_error(&1)))
+    |> Enum.map(&(Map.get(&1, "message")))
+    |> Enum.join("\n")
+  end
+
+  defp fail_if_webdriver_selenium(function) do
+    if false == Enum.any?(["phantomjs", "chrome_driver"], fn(x) -> x == System.get_env("WEBDRIVER") end) do
+        raise Hound.NotSupportedError, function: function
+    end
+  end
+
+  defp get_browser_log() do
+    session_id = Hound.current_session_id
+    make_req(:post, "session/#{session_id}/log", %{type: "browser"})
+  end
+
+  defp is_error(map) do
+	  (level(map, "WARNING") || level(map, "CRITICAL"))
```

Comment:
Thanks. Actually it seems there was a tab there, and github takes tabs as 4 spaces.. Changed to spaces.. Same thing a few other places.. Need to setup my emacs to write two spaces when i press tab :)